target/linux/ramips/image/mt7621.mk

```diff
@@ -9,6 +9,10 @@ DEFAULT_SOC := mt7621
 KERNEL_DTB += -d21
 DEVICE_VARS += UIMAGE_MAGIC SERCOMM_HWNAME
 
+define Build/asus-factory
+        $(STAGING_DIR_HOST)/bin/mkasusfw $(1) $(2) $@
+endef
```

Comment:
It's not necessary to keep this twice, just move it to image/Makefile when you touch it in the first patch of your series (and add a very short comment about that).

Reply:
Done, moved into image/Makefile, removed from other files